src/dotnet-format.csproj

```diff
@@ -52,12 +52,18 @@
     <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="$(MicrosoftExtensionsFileSystemGlobbingVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" />
 
+    <!-- For easy testing of analyzer references
```

Comment:
I would prefer a more complete solution than this. not blocking but I would remove this before checking in